src/data/text/species_names.h

```diff
@@ -1068,7 +1093,9 @@ const u8 gSpeciesNames[][POKEMON_NAME_LENGTH + 1] = {
     [SPECIES_ROTOM_FROST] = _("Rotom"),
     [SPECIES_ROTOM_MOW] = _("Rotom"),
     [SPECIES_ROTOM_WASH] = _("Rotom"),
-    // Giratina
+	// Origin
```

Comment:
```suggestion
    // Origin
```

Reply:
// Origin Forme instead?
   Everything else is written as Origin Forme. i think

Reply:
Sure.